qiita_db/metadata_template/test/test_sample_template.py

```diff
@@ -238,50 +239,47 @@ class BaseTestSampleTemplate(TestCase):
     def _set_up(self):
         self.metadata_dict = {
             'Sample1': {'physical_specimen_location': 'location1',
-                        'physical_specimen_remaining': True,
-                        'dna_extracted': True,
+                        'physical_specimen_remaining': 'true',
+                        'dna_extracted': 'true',
                         'sample_type': 'type1',
-                        'collection_timestamp':
-                        datetime(2014, 5, 29, 12, 24, 51),
+                        'collection_timestamp': '05/29/14 12:24:15',
                         'host_subject_id': 'NotIdentified',
                         'Description': 'Test Sample 1',
                         'str_column': 'Value for sample 1',
-                        'int_column': 1,
-                        'latitude': 42.42,
-                        'longitude': 41.41,
-                        'taxon_id': 9606,
+                        'int_column': '1',
```

Comment:
The previous PR change the populate and the test that failed but not the creation/updates within tests as the constrains were not there.